muacrypt/recommendation.py

```diff
@@ -41,13 +44,16 @@ def ui_recommendation(self):
         if (pre == 'available' and
                 self.prefer_encrypt == 'mutual' and
                 self.peer.prefer_encrypt == 'mutual'):
-            return 'encrypt'
+            # check that the last message had an AC header
+            if self.peer._latest_msg_entry().keydata:
+                return 'encrypt'
```

Comment:
Is this from the AC spec?

Reply:
yes -- see the tests. if you had mutual encrypt and then a non-ac mail came in, you woul still recommend "encrypt" instead of "available".

Reply:
so there is discussion currently if  to default to "encrypt" even if we last saw cleartext mails. 

For now, i'd like to go for available -- and if L1 does not change, state this as a deviation.  E.g. the bot-mutual@autocrypt.org would for 35 days send encrypted mail if it saw a mutual-AC mail from an address. i don't think that's reasonable.